src/actors/builtin/codes.go

```diff
@@ -0,0 +1,49 @@
+package builtin
+
+import (
+	"github.com/filecoin-project/specs/actors/abi"
+	"github.com/ipfs/go-cid"
+	mh "github.com/multiformats/go-multihash"
+)
+
+// The built-in actor code IDs
+var SystemActorCodeID abi.ActorCodeID
+var InitActorCodeID abi.ActorCodeID
+var CronActorCodeID abi.ActorCodeID
+var AccountActorCodeID abi.ActorCodeID
+var StoragePowerActorCodeID abi.ActorCodeID
+var StorageMinerActorCodeID abi.ActorCodeID
+var StorageMarketActorCodeID abi.ActorCodeID
+var PaymentChannelActorCodeID abi.ActorCodeID
+var MultisigActorCodeID abi.ActorCodeID
+
```

Comment:
do we need to include BurntFundActor and RewardActor?

Reply:
BurntFunds is just an account. Will add Reward